packages/gatsby-source-shopify/README.md

```diff
@@ -76,8 +76,8 @@ module.exports = {
     {
       resolve: "gatsby-source-shopify",
       options: {
-        password: process.env.SHOPIFY_ADMIN_PASSWORD,
-        storeUrl: process.env.SHOPIFY_STORE_URL,
+        password: process.env.SHOPIFY_SHOP_PASSWORD,
+        storeUrl: process.env.GATSBY_SHOPIFY_STORE_URL,
```

Comment:
Is this needed client side?

Reply:
🤔 I guess it makes sense that it would be needed for cart redirection and other client side Shopify API stuff 👍

Reply:
The store URL? Yeah it's used client side in the shopify starter. You use it alongside your `STOREFRONT_ACCESS_TOKEN` to do queries from the client.

Reply:
There's no harm to expose that even if they don't access the shop clientside, it just makes for a less confusing experience if we use the same naming convention across all our shopify docs

Reply:
Nice, makes sense, thanks for the explanation!